src/integration-test/java/org/ow2/proactive/workflow_catalog/rest/controller/WorkflowControllerIntegrationTest.java

```diff
@@ -74,6 +74,10 @@
 
     private static final String WORKFLOW_RESOURCE = "/buckets/{bucketId}/workflows/{workflowId}";
 
+    private static final String LAYOUT =
```

Comment:
I would have renamed all references to layout name to a new name like _layoutMetadata_ to better assess there is nothing related to a UI element associated to the workflow catalog.